packages/dataviews/src/components/dataviews/stories/fixtures.tsx

```diff
@@ -682,6 +682,10 @@ export const fields: Field< SpaceObject >[] = [
 		id: 'satellites',
 		type: 'integer',
 		enableSorting: true,
+		filterBy: {
+			operators: [ 'is', 'isNot' ],
```

Comment:
This is another thing I've run into with the date filter. Consumers shouldn't need to declare this just because an `integer` doesn't support `isAny` or `isNone`, it should be the field type that provides a list of "supported operators". As a first step, I've added the logic to the `sanitizeOperators` in the date filter PR, but I think it could be part of the filed type definition itself.

Reply:
Another question is: what does it mean that a field of type `integer` provides a list of `elements`? Do we ignore them? Or is it a way to provide "presets"?

Reply:
> but I think it could be part of the filed type definition itself.

Sounds good. I'll explore the idea of adding a list of supported operators to the field type definition. 👍

> Another question is: what does it mean that a field of type integer provides a list of elements? Do we ignore them? Or is it a way to provide "presets"?

It's a way to provide "presets". We still allow consumers to declare a list of elements for a field of type integer, like a text field does.

Reply:
I've update the PR with the following changes:

- The `userInput` prop has been removed and the field type is now used to determine if the filter should be a user input.
- The operators logic has been moved from `sanitizeOperators` to the field type definition.
- The control is now rendered using the `Edit` field.

Could you please take a look and let me know what you think?